data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -88,10 +88,8 @@
   "difficulty": 2,
   "time": 10000,
   "book_learn": [["adv_chemistry", 1] , ["textbook_chemistry", 1]],
-  "tools": [
-    [
-      [ "chemistry_set", 5 ]
-    ]
+  "qualities": [
+    {"id":"CHEM","level":1}
```

Comment:
I based the chemistry set on the standard chemistry set as listed by Wikipedia. The set seems to have been designed to be at least partially DIY and accessible as learning tools for school kids (maybe not elementary school, but other than that it was fair game).  It does not include a source of UV light but we can handwave it, I guess?